docs/running-on-mesos.md

```diff
@@ -516,6 +516,15 @@ See the [configuration page](configuration.html) for information on Spark config
     Fetcher Cache</a>
   </td>
 </tr>
+<tr>
+  <td><code>spark.mesos.unavailabilityThreshold</code></td>
```

Comment:
Is the purpose of `unavailabilityThreshold` to run on the machine before the maintenance begins?